### ISSUE TITLE
may you share more git hubs links to  Learn how to install and use  open-source extensions in my sageamker code deploy endpoint

### ISSUE THREAD

**Issue by Sandy4321** [seanpmorgan/studio-notebook-webinar#1]
great code and video
https://www.youtube.com/watch?v=1iSiN4sVMjE
especially 
Objective 2: Learn how to install open-source extensions

may you share more git hubs links to  Learn how to install and use  open-source extensions in my sageamker code deploy endpoint 
predictor = tuner.deploy(initial_instance_count=1, instance_type="ml.m5.xlarge")

lets say I want to use  CDClassifier , which needs to be installed to built in scikit or tensor flow docker images of sageamker 
https://github.com/scikit-learn-contrib/lightning
from sklearn.datasets import fetch_20newsgroups_vectorized
from lightning.classification import CDClassifier

# Load News20 dataset from scikit-learn.
bunch = fetch_20newsgroups_vectorized(subset="all")
X = bunch.data
y = bunch.target

# Set classifier options.
clf = CDClassifier(penalty="l1/l2",
                   loss="squared_hinge",
                   multiclass=True,
                   max_iter=20,
                   alpha=1e-4,
                   C=1.0 / X.shape[0],
                   tol=1e-3)

# Train the model.
clf.fit(X, y)

# Accuracy
print(clf.score(X, y))

# Percentage of selected features
print(clf.n_nonzero(percentage=True))
